pkg/runutil/runutil.go

```diff
@@ -1,13 +1,13 @@
+// Package runutil provides helpers to process control
```

Comment:
```suggestion
// Package runutil provides helpers to advanced function scheduling control like repeat or retry.
```

Reply:
We can ignore remove this ` and log every errors even if it is a tiny error.` - it's quite misleading and too detailed for package description maybe (: